connection.go

```diff
@@ -134,10 +139,20 @@ func (d defaultLogger) Report(event ConnLogKind, conn *Connection, v ...interfac
 // always returns array of array (array of tuples for space related methods).
 // For Eval* and Call* Tarantool always returns array, but does not forces
 // array of arrays.
+//
+// If connected to Tarantool 2.10 or newer and WatchersFeature is required,
+// connection supports server graceful shutdown. In this case, server will
```

Comment:
`WatchersFeature is required`
Why do we need this restriction? This feature might just work.

Reply:
> Why do we need this restriction?

Well, we already have a restriction for user's watchers which is exactly the same. And I think it's a bit more predictable this way: otherwise we won't have a way to distinct "watcher hasn't been created because watchers are not supported" and "we failed to create the watcher because something went wrong (and we completely had ignored the reason)".

Reply:
> "watcher hasn't been created because watchers are not supported"

We could check conn.serverProtocolInfo.Features for that purpose.

Reply:
> We could check conn.serverProtocolInfo.Features for that purpose and just create or not the watcher.

Then why don't we do the same for user watchers? I think consistency will be nice here (in terms of having a simple code too -- otherwise we'll need to create additional "create a watcher even is Watchers are not required" internal handle).

Reply:
> Then why don't we do the same for user watchers?

Make less difference with the pool  from the user size.

> I think consistency will be nice here (in terms of having a simple code too -- otherwise we'll need to create additional "create a watcher even is Watchers are not required" internal handle).

This is an internal details, the user does not really need to worry about such details in case of graceful shutdown.

Reply:
Ok, changed the condition